lib/workers/pr/changelog/release-notes.ts

```diff
@@ -260,6 +260,29 @@ export async function getReleaseNotesMd(
   return null;
 }
 
+/**
+ * Determine how long to cache release notes based on when the version was released.
+ *
+ * It's not uncommon for release notes to be updated shortly after the release itself,
+ * so only cache for about an hour when the release is less than a week old. Otherwise,
+ * cache for days.
+ */
+function releaseNotesCacheMinutes(releaseDate?: string | Date): number {
+  // for an invalid or missing release date, default to "now"
+  const releaseTS = new Date(releaseDate || '').valueOf() || Date.now();
```

Comment:
Use DateTime from luxon.

Reply:
Ah if Luxon is already a dep, I'd be happy to use it instead. I'll change that now.

Reply:
Of course there will still be slop with Luxon since we aren't positive what the provided arg is (ISO string, JS Date instance, or undefined).